integration-cli/docker_cli_daemon_test.go

```diff
@@ -362,7 +359,14 @@ func (s *DockerSuite) TestDaemonIPv6FixedCIDR(c *check.C) {
 		c.Fatalf("Container should have a global IPv6 address")
 	}
 
-	// TODO: Check IPv6 def gateway in inspect o/p (once docker/docker 19001 is merged
+	out, err = s.d.Cmd("inspect", "--format", "'{{.NetworkSettings.Networks.bridge.IPv6Gateway}}'", "ipv6test")
+
+	if err != nil {
+		c.Fatalf("Error inspecting container: %s, %v", out, err)
+	}
+
+	c.Assert(strings.Trim(out, " \r\n'"), checker.Equals, "2001:db8:2::100", check.Commentf("Container should have a global IPv6 gateway"))
```

Comment:
That's exactly what we want no?
TrimSpace is the proper function to use here, especially since it will handle other cases as well in case for some reason the output changes.